test/cmdline_test.py

```diff
@@ -84,6 +84,17 @@ class FooSubClass(FooBaseClass):
     pass
 
 
+class LoggingTask(luigi.Task):
+
+    def complete(self):
+        return False
+
+    def run(self):
+        import logging
+        logging.getLogger().addHandler(logging.StreamHandler())
```

Comment:
Can we have a comment here that this test is actually testing a malpractice, i.e. that https://docs.python.org/2/library/logging.html#logger-objects should be comprehended and `logging.getLogger(__name__)` should be used? Then +1.